include/rocksdb/options.h

```diff
@@ -931,6 +931,7 @@ struct DBOptions {
   // Default: null
   std::shared_ptr<WriteBufferManager> write_buffer_manager = nullptr;
 
+  // DEPRECATED
   // Specify the file access pattern once a compaction is started.
   // It will be applied to all input files of a compaction.
   // Default: NORMAL
```

Comment:
Like #5431, you can add a comment like:

```
  // This flag has no effect on the behavior of compaction and we plan to delete
  // it in the future.
```

Then modify `BlockBasedTable::SetupForCompaction()` to do nothing.